spec/features/photo_feature_spec.rb

```diff
@@ -0,0 +1,48 @@
+require 'rails_helper'
+
+feature 'Photos' do
+  context 'No photos have been added' do
+    scenario 'Should display a prompt to add a restaurant' do
+      visit '/photos'
+      expect(page).to have_content 'No photos yet'
+      expect(page).to have_link 'Add a photo'
+    end
+  end
+
+  context 'Photos have been added' do
+    before do
+      Photo.create(descr: 'Mount Fuji')
+    end
+
+    scenario 'Display photos' do
+      visit '/photos'
+      expect(page).to have_content('Mount Fuji')
+      expect(page).not_to have_content('No photos yet')
+    end
+  end
+
+  context 'Adding photos' do
+    scenario 'prompts user to add photo, then displays the description' do
+      visit '/photos'
+      click_link 'Add a photo'
+      fill_in 'Description', with: 'Mount Fuji'
+      click_button 'Add photo'
+      expect(page).to have_content 'Mount Fuji'
+      expect(current_path).to eq '/photos'
+    end
+  end
+
+  context 'viewing photos' do
+
+  let!(:mount_fuji){Photo.create(descr:'Mount Fuji')}
```

Comment:
Space missing after colon.<br>Space missing to the left of {.<br>Space missing inside {.<br>Space missing inside }.